### PR TITLE
Inspec 4 warning

### PR DESCRIPTION
**Pull request by micheelengronne**
Inspec 4 uses value instead of default

### CODE REVIEW

**Comment by micheelengronne** [comment]
It fails because of a rubocop syntax update.

**Comment by micheelengronne** [comment]
`bundler: command not found: inspec`

Ok, the CI needs debugging.

**Comment by chris-rock** [comment]
I think we need to pin to `gem 'inspec', '~> 3'`. Due to licensing we cannot use inspec 4.

**Comment by micheelengronne** [comment]
I know the license problem.

But, Inspec 3 is not maintained anymore.

The chef license concerns the `inspec-bin` gem only.

Inspec source code is still opensource (Apache2) as well as the `inspec` gem.

If you compile yourself a binary (or a script) to use the `inspec` gem you are not subject to the chef license.

I think we should maintain the profile for Inspec 4 as Inspec 3 is EOL.

**Comment by chris-rock** [comment]
We stay backwards-compatible with InSpec 3 for now. Your change works in InSpec 3 too. Once ready, dev-sec will switch to https://github.com/cinc-project/auditor

**Comment by micheelengronne** [comment]
Thanks a bunch for auditor. I didn't know it. I compile my own gem. I will give it a try.

**Comment by chris-rock** [comment]
Until then I recommend to just stick with InSpec 3

**Comment by micheelengronne** [comment]
Ok, I see that the gem is currently not maintained https://rubygems.org/gems/cinc-auditor-bin and does not follow the Inspec rythm. I will stick with my own compiled Inspec gem and try to make backward compatible contributions.

**Comment by chris-rock** [comment]
@micheelengronne I fixed the CI in latest master, can you please rebase?

File: Gemfile [hunk 1]
```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
```
Comment (chris-rock) [inline Gemfile]: I adapted the robo config to avoid adding this header. We should remove them.

**Comment by micheelengronne** [comment]
Thanks a lot. Can you do the corresponding release ? I like SemVer :)

**Pull request merged**